test/SlowTests/Server/Documents/ETL/SQL/SqlEtlTests.cs

```diff
@@ -1179,6 +1180,18 @@ await session.StoreAsync(new Order
             }
         }
 
+        [RequiresMsSqlRetryFact(delayBetweenRetriesMs: 1000)]
+        public async Task CanTestConnection()
```

Comment:
This test will _always_ pass. There is no `await`. I don't think you need it to be asynchronous

Reply:
Oh, no it's not `async void` issue. Anyway you don't need `async` here at all

Reply:
ran the test before pushing in both fail and success scenario - confirming it's working 
will remove async though :]

Reply:
changed to `public void`